src/components/RNMarkdownTextInput.tsx

```diff
@@ -63,7 +63,7 @@ function RNMarkdownTextInputWithRef({maxLength, parser, ref, forwardedFSClass =
     );
 
     useEffect(() => {
-        runOnLiveMarkdownRuntime(() => {
+        scheduleOnLiveMarkdownRuntime(() => {
             'worklet';
 
             mentionsSharedVal.set(availableLoginsList);
```

Comment:
Ideally we'd like `scheduleOnLiveMarkdownRuntime` to have the same signature as `scheduleOnUI` does, i.e. without the need to call the returned function, like this:
```diff
-scheduleOnLiveMarkdownRuntime(fn)(...args); // currently you need to add `();` at the end
+scheduleOnLiveMarkdownRuntime(fn, ...args);
```